static/app/views/issueList/overviewFc.tsx

```diff
@@ -704,14 +672,6 @@ function IssueListOverviewFc({
     // reload data.
     if (!isEqual(previousRequestParams, requestParams)) {
       fetchData(selectionChanged);
-    } else if (
```

Comment:
I don't think this is necessary anymore, it should react to the URL changing now